template/app/build.gradle.kts

```diff
@@ -110,6 +110,13 @@ android {
             isIncludeAndroidResources = true
             isReturnDefaultValues = true
         }
+        unitTests.all {
+            if (it.name != "testStagingDebugUnitTest") {
+                it.extensions.configure(kotlinx.kover.api.KoverTaskExtension::class) {
+                    isDisabled.set(true)
+                }
+            }
+        }
```

Comment:
Should we add this in the `data` module too? 💭

Reply:
Also, on [this line](https://github.com/nimblehq/android-templates/pull/344/files#diff-8d83febc7bad9355f66dcdf21f1a25e2b5ee979000e8b11b4446d880542f969bR59-R63), there's a comment regarding Jacoco. 

Would you mind re-checking this? 🦖